app/src/main/java/com/itsaky/androidide/utils/EditorDecorationBridge.kt

```
@@ -0,0 +1,113 @@
+            val app = BaseApplication.baseInstance
+            lastNightMode = app.resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK
+            app.registerComponentCallbacks(object : ComponentCallbacks {
+                override fun onConfigurationChanged(newConfig: Configuration) {
+                    val night = newConfig.uiMode and Configuration.UI_MODE_NIGHT_MASK
+                    if (night != lastNightMode) {
+                        lastNightMode = night
+                        refresh()
+                    }
+                }
+
+                override fun onLowMemory() {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)